src/core/lib/iomgr/timer_generic.c

```diff
@@ -92,13 +113,15 @@ void grpc_timer_list_init(gpr_timespec now) {
   g_initialized = true;
   gpr_mu_init(&g_mu);
   g_clock_type = now.clock_type;
+  g_start_time = now;
+  g_min_timer = timespec_to_atm(now);
 
   for (i = 0; i < NUM_SHARDS; i++) {
     shard_type *shard = &g_shards[i];
     gpr_mu_init(&shard->mu);
     grpc_time_averaged_stats_init(&shard->stats, 1.0 / ADD_DEADLINE_SCALE, 0.1,
                                   0.5);
-    shard->queue_deadline_cap = now;
+    shard->queue_deadline_cap = timespec_to_atm(now);
```

Comment:
It's debatable, but I find this clearer: I don't need to trace more variables through the function to see that it's doing the right thing (and given this is absolutely not critical path that's a win for me).